hdate/hdate_julian.py

```diff
@@ -36,10 +36,13 @@ def _days_from_3744(hebrew_year):
     week_day = parts_left_in_week / PARTS_IN_DAY
 
     # Special cases of Molad Zaken
-    if ((leap_left < 12 and week_day == 3 and
-         parts_left_in_day >= get_chalakim(9 + 6, 204)) or
-            (leap_left < 7 and week_day == 2 and
-             parts_left_in_day >= get_chalakim(15 + 6, 589))):
+    if (leap_left < 12 and week_day == 3 and
```

Comment:
this is a dangerous change,
if somehow we will change the if order we will have a bug,
I think it's better in the old code